ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -183,18 +174,64 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
     }
 
     // Reference temperature
-    const auto& reference_temperature =
+    double const reference_temperature =
         //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__reference_temperature}
         config.getConfigParameter<double>(
             "reference_temperature", std::numeric_limits<double>::quiet_NaN());
+    DBUG("Use '%s' as reference temperature.", "reference_temperature");
+
+    //Specific gas constant
+    double const specific_gas_constant =
+        //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__specific_gas_constant}
+        config.getConfigParameter<double>(
+            "specific_gas_constant", std::numeric_limits<double>::quiet_NaN());
+    DBUG("Use '%s' as specific gas constant.", "specific gas constant");
+
+    // Fluid compressibility
+    double const fluid_compressibility =
+        //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__fluid_compressibility}
+        config.getConfigParameter<double>(
+            "fluid_compressibility", std::numeric_limits<double>::quiet_NaN());
+    DBUG("Use '%s' as fluid compressibility parameter.", "fluid_compressibility");
```

Comment:
```suggestion
    DBUG("Use '%f' as fluid compressibility parameter.", fluid_compressibility);
```

Reply:
In all other DBUG messages the name of the parameter is given, not the value, so I would leave it here the same way.

Reply:
Okay, if you want output the parameter name and it is 'hard coded' you could put it directly in the message.
```suggestion
    DBUG("Use 'fluid_compressibility' as fluid compressibility parameter.");
```